tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This test is failing for me when trying to run:
TEST-UNEXPECTED-ERROR | test_settings_NC_RTL.py TestSettingsRTL.test_settings_app | TimeoutException: TimeoutException: Timed out after 10.1 seconds


Traceback (most recent call last):
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_client-0.16-py2.7.egg/marionette/marionette_test.py", line 296, in run
    testMethod()
  File "/Users/mwargers/B2G/gaia_clean/tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py", line 61, in test_settings_app
    settings.return_to_prev_menu(settings.screen_element)
  File "/Users/mwargers/B2G/gaia_clean/tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py", line 356, in return_to_prev_menu
    Wait(self.marionette).until(expected.element_displayed(parent_view))
  File "/Users/mwargers/.virtualenvs/test3/lib/python2.7/site-packages/marionette_driver-0.9-py2.7.egg/marionette_driver/wait.py", line 143, in until
    cause=last_exc)
TEST-INFO took 140866ms

SUMMARY
-------
passed: 0
failed: 1
todo: 0